### ISSUE TITLE
Wikidata links missing from some entities, salience score from all

### ISSUE THREAD

**Issue by darklajid** [rosette-api/csharp#91]
Using the latest version of the API and my test key, I send a document to the Entity extraction endpoint.

I enable all features (testing coverage, evaluating Rosette atm) using

    entitiesCAPI.SetOption("calculateConfidence", true);
    entitiesCAPI.SetOption("calculateSalience", true);
    entitiesCAPI.SetOption("includeDBpediaType", true);
    entitiesCAPI.SetOption("linkEntities", true);

Results include

````
{"mention":"JAPAN","normalized":"JAPAN","entityId":"T5","type":"LOCATION","count":8,"confidence":0.85243988}
{"mention":"SWITZERLAND","normalized":"Switzerland","entityId":"Q39","type":"LOCATION","count":4,"confidence":0.93196756,"dbpediaType":"Place/PopulatedPlace/Country"}
````

1) Why a temporary ID for Japan? I have a lot more entities with a T ID (say, Tokyo, some more examples of nonworking results below)

2) Why no salience score for any value?

More missing wikidata links in my samples:

````
{"mention":"TOKYO","normalized":"TOKYO","entityId":"T10","type":"LOCATION","count":3,"confidence":0.95691532}
{"mention":"FRIBOURG","normalized":"FRIBOURG","entityId":"T13","type":"LOCATION","count":1,"confidence":0.18674469}
````

**Comment by darklajid** [comment]
Digging a bit it seems that https://github.com/rosette-api/csharp/blob/develop/rosette_api/EntitiesResponse.cs doesn't have any notion of salience. Using the raw Rest API I can get these values, but this c-sharp wrapper fails to expose the field?

**Comment by seth-mg** [comment]
Hi @darklajid
We released a new version of the binding, 1.12.2, that includes mention offsets, linking confidence and salience in the response data for `/entities` and `/sentiment` calls.

Thanks for reporting this.
